### PR TITLE
Added support for Service Alerts in Realtime

### PR DESCRIPTION
**Pull request by AvidDabbler**
I was looking to use your library, but specifically wanted Service Alerts. Let me know if this fits inline with how you would structure it.

example service alert => https://developers.google.com/transit/gtfs-realtime/examples/alerts

### CODE REVIEW

File: src/realtime.d.ts [hunk 1]
```diff
@@ -68,7 +65,65 @@ export interface Entity {
...
+interface active_period {
+  start?: Date;
+  end?: Date;
+}
+interface TripDescriptor {
+  trip_id: string;
+  start_time?: string;
+  end_time?: string;
+}
+interface informed_entity_route {
```
Comment (k-yle) [inline src/realtime.d.ts]: could we use PascalCase for the names of interfaces please, just to be consistent with the existing interfaces. 

e.g. `informed_entity_route` -> `InformedEntityRoute`

File: src/realtime.d.ts [hunk 2]
```diff
@@ -68,7 +65,65 @@ export interface Entity {
   vehicle?: VehicleUpdate;
   is_deleted: boolean;
 }
-
+interface active_period {
+  start?: Date;
```
Comment (k-yle) [inline src/realtime.d.ts]: my only concern is that these aren't really dates, they're numbers. 

If you tell typescript that `start` is a `Date`, it suggests that you can use Date methods like `start.toISOString()` or `start.toLocaleDateString()` which would actually crash if you tried it, beacuse `start` is a number

I'm interested in what you think here - I'm not 100% sure what the correct way of doing it is

**Review by k-yle — CHANGES_REQUESTED** [review]
thanks for the PR! I've left some minor comments/questions

File: src/realtime.d.ts (hunk 2 above)
Comment (AvidDabbler) [reply src/realtime.d.ts]: Oops you are right! Sorry about that I caught the error on one of them but must have missed these

**Review by k-yle — APPROVED** [review]

**Comment by k-yle** [comment]
thanks for the PR! this is released in `v2.2.0`

**Pull request merged**